core/test/audits/byte-efficiency/byte-efficiency-audit-test.js

```diff
@@ -398,4 +409,19 @@ describe('Byte efficiency base audit', () => {
     const result = await MockAudit.audit(artifacts, {settings, computedCache});
     expect(result.details.overallSavingsMs).toBeCloseTo(575, 1);
   });
+
+  describe('#scoreForWastedMs', () => {
```

Comment:
`scoreForWastedMs` is imported and used independently of audits extending ByteEfficiencyAudit in at least `prioritize-lcp-image`, so it felt worth it to test separately as well.